tests/modules/nf-core/vgan/haplocart/main.nf

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { VGAN_HAPLOCART } from '../../../../../modules/nf-core/vgan/haplocart/main.nf' addParams( options: [args: ''] )
```

Comment:
```suggestion
include { VGAN_HAPLOCART } from '../../../../../modules/nf-core/vgan/haplocart/main.nf'
```

Reply:
See above about latest `tools` template - this is another example of out of date.